deeprank_gnn/models/error.py

```diff
@@ -1,5 +1,3 @@
-
-
-class UnknownAtomError(ValueError):
-    "should be raised when an unknown atom type is encountered"
-    pass
+class UnknownAtomError(ValueError):

```

Comment:
we should either implement something here or remove this class everywhere.

Reply:
In deeprank, we take a specific action when this specific error is thrown. All other errors are considered unexpected.
We need to know when an unknown atom is encountered, so that we can skip it.
It doesn't matter whether this class implements something, as long as it can be identified as an unknown atom error.

Reply:
What about printing the ValueError?

Reply:
You mean that we should let python check the error message in the catch block?

Reply:
Sorry, I didn't realise that ValueError is the parent class of UnknownAtomError. I'll insert [an init method](https://www.programiz.com/python-programming/user-defined-exception) to the class with the explicit inheritance with super(), just for completeness.